NeoMathEngine/src/CPU/CpuMathEngineDnn.cpp

```diff
@@ -1190,16 +1084,12 @@ void CCpuMathEngine::LinearInterpolation( const CConstFloatHandle& dataHandle, c
 	const int dataBatchStep = scaledAxis * objectSize;
 	const int resultBatchStep = static_cast<int>( scale * scaledAxis ) * objectSize;
 
-	const int opCount = objectCount * resultBatchStep;
-	const int curThreadCount = IsOmpRelevant( objectCount, opCount ) ? threadCount : 1;
-	CFloatHandleStackVar buff( *this, objectSize * curThreadCount );
-
+	CFloatHandleStackVar buff( *this, objectSize );
 	TCoordTransformer getCoords = getCoordTransformer( coords );
 	TCoordRound roundCoords = getCoordRound( round );
 
-	NEOML_OMP_FOR_NUM_THREADS( curThreadCount )
 	for( int b = 0; b < objectCount; ++b ) {
-		float* currBuff = GetRaw( buff.GetHandle() ) + OmpGetThreadNum() * objectSize;
+		float* currBuff = GetRaw( buff.GetHandle() ) + objectSize;
```

Comment:
`buff` contains `objectSize` elements.
`GetRaw( buff.GetHandle() ) + objectSize` -> out of bounds